src/public/translations/en-SG/landing.json

```diff
@@ -110,7 +110,7 @@
     }
   },
   "FOOTER": {
-    "APP_NAME": "FormSG",
+    "APP_NAME": "Form",
```

Comment:
are we really doing this here? even in the sg version?

Reply:
it's not even gonna look like a brand, it's just gonna look like the word "Form" lol

Reply:
Doing this according to the design haha 🤷

Reply:
am I the only one who thinks the word Form in "**Form** _Build government forms in minutes_" just looks like the word Form and not a brand

Reply:
@pearlyong thoughts?

Reply:
can we see it in staging to have a better sense? 

Go has 'Go.gov.sg' in both the website title and in the footer; the other products have mostly identifiable brand names.